string path_to_tmp_img
float32 maxSimVal
---
